app.js

```diff
@@ -1,11 +1,136 @@
 /* Imports */
 // this will check if we have a user and set signout link if it exists
 import './auth/user.js';
+import { createItem, getListItems, gotIt, renderItem, removeItems, deleteCompleted } from './fetch-utils.js';
 
 /* Get DOM Elements */
 
+const form = document.querySelector('.new-form');
+const deleteItem = document.getElementById('delete');
+const listEl = document.querySelector('.list');
+const errorDisplay = document.getElementById('error-display');
+const deleteComplete = document.getElementById('delete-complete');
+
 /* State */
 
+let items = [];
+let error = null;
+
 /* Events */
+window.addEventListener('load', async () => {
+    const response = await getListItems();
+    error = response.error;
+    items = response.data;
+
+    if (error) {
+        displayError();
+    }
+
+    if (items) {
+        displayItems();
+    }
+});
+
+async function populateItems() {
+    const response = await getListItems();
+    error = response.error;
+    items = response.data;
+
+    if (error) {
+        displayError();
+    }
+
+    if (items) {
+        displayItems();
+    }
+}
+
+form.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    
+    const formData = new FormData(form);
+    const newItem = {
+        item: formData.get('item'),
+        quantity: formData.get('quantity'),
+    };
+
+    const response = await createItem(newItem);
+    error = response.error;
+    const item = response.data;
+    // renderItem(item);
+
+    if (error) {
+        displayError();
+    } else {
+        items.push(item);
+        displayItems();
+        form.reset();
+        populateItems();
+    }
```

Comment:
i think this might be your issue with displaying undefined, not sure how to fix it but i'm noticing if there's no error you're calling ```displayItems()``` and then ```populateItems()``` and ```populateItems()``` is calling ```displayItems()``` inside of it. i think you might be able to get rid of line 66